osf/models/registrations.py

```diff
@@ -666,6 +666,14 @@ def flags(self):
     def flags(self, flags):
         self._metaschema_flags.update(flags)
 
+    @property
+    def branched_from_type(self):
+        if type(self.branched_from) == DraftNode:
```

Comment:
typically we want to use `isinstance` to check this. [For complicated reasons](https://stackoverflow.com/questions/1549801/what-are-the-differences-between-type-and-isinstance)

Something like this maybe?
```
if isinstance(self.branched_from, (DraftNode, Node)):
   return self.__name__
else:
   return ''
```